account_vat_period_end_statement/models/account.py

```diff
@@ -759,16 +759,27 @@ def _get_credit_debit_lines(self, statement):
                 ("type_tax_use", "in", ["sale", "purchase"]),
             ]
         )
+        if statement.account_ids:
+            taxes = taxes.filtered(
+                lambda tax: tax.vat_statement_account_id in statement.account_ids
+            )
         for tax in taxes:
-            if (
-                tax.vat_statement_account_id.id in statement.account_ids.ids
-                or not statement.account_ids
+            # se ho una tassa padre con figli cee_type, condidero le figlie
```

Comment:
Esiste ancora `cee_type`?

Reply:
> Esiste ancora `cee_type`?

In effetti è stato rimosso proprio dal tuo commit [https://github.com/OCA/l10n-italy/commit/df780c4469cf8ae734d4ce8e25cf5eea84acdfac](url)

Reply:
@jado95 puoi sistemare? siamo molto vicini al merge :)

Reply:
@jado95 ping